### PR TITLE
Release notes draft for 6.6.0

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -24,6 +26,97 @@ This section summarizes the changes in the following releases:
...
+* BUGFIX: Add option for PQ checkpoint retry https://github.com/elastic/logstash/pull/10246[#10246]
+* Add documentation for java-execution flag https://github.com/elastic/logstash/pull/10248[#10248]
+* BUGFIX: Properly set thread name in thread context https://github.com/elastic/logstash/pull/10272[#10272]
+* Announcement: HTTP lookup filter and Memcached get/set filter now GA.
+
+==== Plugins
+
+*Aggregate Filter*
+
+* new feature: add ability to dynamically define a custom `timeout` or `inactivity_timeout` in `code` block (fix issues [#91](https://github.com/logstash-plugins/logstash-filter-aggregate/issues/91) and [#92](https://github.com/logstash-plugins/logstash-filter-aggregate/issues/92))
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: The MD link format doesn't render correctly in asciidoc. Should be:
https://github.com/logstash-plugins/logstash-filter-aggregate/issues/91[#91]

The links in the Logstash section are correct.  Only the plugin links are affected. (I didn't flag them all, but can if that would be helpful.)

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -24,6 +26,97 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-6-0]]
+=== Logstash 6.6.0 Release Notes
+
+
+=== Logstash Pull Requests with label v6.6.0
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc]: The doc won't build as is.  I'm guessing that asciidoc doesn't like two headings of the same level without text between.  I got it to build when I deleted `=== Logstash Pull Requests with label v6.6.0`

Comment (guyboertje) [reply docs/static/releasenotes.asciidoc]: Will remove

**Comment by guyboertje** [comment]
Single test failure is unrelated.

**Comment by colinsurprenant** [comment]
LGTM

**Pull request merged**